app/src/main/java/com/github/se/eventradar/viewmodel/EventsOverviewViewModel.kt

```diff
@@ -288,6 +288,11 @@ constructor(
     // Reset search and filter when tab is changed
     state.value =
```

Comment:
you should always be using state.update {} to ensure atmocity and avoid race conditions, I made a PR which changed this for all VMs so you do not need to make this change

Reply:
Great thanks, noted.